content-src/experiments/activity-stream.yaml

```diff
@@ -18,6 +18,9 @@ xpi_url: 'https://moz-activity-streams.s3.amazonaws.com/dist/activity-streams-la
 addon_id: '@activity-streams'
 gradient_start: '#0996F8'
 gradient_stop: '#0675D3'
+min_release:
```

Comment:
I mean, these fields don;t need to be here...i just figured we could keep the yaml files consistent